src/test/Test.FAKECore/ReleaseNotesSpecs.cs

```diff
@@ -132,7 +132,7 @@ public class when_parsing_many_pre_release_versions
 #### 2.0.0-alpha001 - December 15 2013
 * A";
         static readonly ReleaseNotesHelper.ReleaseNotes expected =
-            ReleaseNotesHelper.ReleaseNotes.New("2.0.0", "2.0.0-rc007", null, new[] { "A" }.ToFSharpList());
+            ReleaseNotesHelper.ReleaseNotes.New("2.0.0", "2.0.0-rc2", null, new[] { "A" }.ToFSharpList());
```

Comment:
The problem is usually the existing packages. But maybe this is less of a
problem now. It was a huge problem couple of years ago before we starter
with paket

On Jul 24, 2016 7:01 PM, "Chester Husk III" <notifications@github.com>
wrote:

> In src/test/Test.FAKECore/ReleaseNotesSpecs.cs
> <https://github.com/fsharp/FAKE/pull/1325#discussion_r71991787>:
>
> > @@ -132,7 +132,7 @@ public class when_parsing_many_pre_release_versions
> >  #### 2.0.0-alpha001 - December 15 2013
> >  * A";
> >          static readonly ReleaseNotesHelper.ReleaseNotes expected =
> > -            ReleaseNotesHelper.ReleaseNotes.New("2.0.0", "2.0.0-rc007", null, new[] { "A" }.ToFSharpList());
> > +            ReleaseNotesHelper.ReleaseNotes.New("2.0.0", "2.0.0-rc2", null, new[] { "A" }.ToFSharpList());
>
> Current logic is wrong, though. Part of the reason I chafed working with
> nuget was because they implemented almost-sorta-but-not-really-semver, so
> none of the existing libraries worked. Do we want to do the same? Besides,
> if people depend on that sorting, they can either
>
>    1. use a prerelease of the form rc.{increment}, where increment is a
>    number and thus compared numerically, or
>    2. use a prerelease of the form rc{padding}{number}, and be
>    consistent, so rc007 compared to rc002,
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/fsharp/FAKE/pull/1325/files/799b4be354b7eb8da836eec694fe7385d43411d7#r71991787>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AADgNEBbyq4YdgDhIJmJc4cPCec9CYrUks5qY5pigaJpZM4JSHRJ>
> .
>